### ISSUE TITLE
Topaz Gigapixels metadata

### ISSUE THREAD

**Issue by barepixels** [MoonRide303/Fooocus-MRE#128]
Topaz Gigapixels output contains metadata which can be seen using XnView.  
![topaz](https://github.com/MoonRide303/Fooocus-MRE/assets/92839033/f46a7401-4f9f-48e2-88e5-119ebdf99e1d)

However, Fooocus-MRE is unable to load Prompt from such files.  Can you look into it?  Can you make Fooocus to use it to load prompt?  If you don't want to it's OK.  Attached is a JPG from Topaz Gigapixels with Fooocus metadata.  I zipped it just incase Google Drive mess with the JPG

https://drive.google.com/file/d/1tQY9bj0WcyxiiYsvDSdLek_kQYfTEmw3/view?usp=sharing

**Comment by MoonRide303** [comment]
Manually extracted prompt from image and saved as json file works fine:
```
{"prompt": "redhead", "negative_prompt": "[deformed | disfigured], poorly drawn, [bad : wrong] anatomy, [extra | missing | floating | disconnected] limb, (mutated hands and fingers), blurry", "styles": [], "real_prompt": ["redhead"], "real_negative_prompt": ["[deformed | disfigured], poorly drawn, [bad : wrong] anatomy, [extra | missing | floating | disconnected] limb, (mutated hands and fingers), blurry"], "seed": 693194942663826797, "width": 512, "height": 768, "sampler": "dpmpp_2m_sde_gpu", "scheduler": "karras", "performance": "Speed", "steps": 30, "switch": 20, "sharpness": 2, "cfg": 7, "base_clip_skip": -2, "refiner_clip_skip": -2, "base_model": "deliberate_v3.safetensors", "refiner_model": "None", "l1": "None", "w1": 1, "l2": "None", "w2": 0.5, "l3": "None", "w3": 0.5, "l4": "None", "w4": 0.5, "l5": "None", "w5": 0.5, "freeu": false, "img2img": false, "revision": false, "positive_prompt_strength": 1, "negative_prompt_strength": 1, "control_lora_canny": false, "control_lora_depth": false, "prompt_expansion": false, "software": "Fooocus 2.0.78.3 MRE"}
```

but attempts to load it from this Topaz-processed file for some reason fails. I need to look into this.

**Comment by MoonRide303** [comment]
Fixed (commit [8640faa](https://github.com/MoonRide303/Fooocus-MRE/commit/8640faaab6420f6a696e6b3a5b289ee1cb0c5839)).

**Comment by barepixels** [comment]
Oh thank you so much.
